SpatialGDK/Documentation/content/project-setup.md

```diff
@@ -0,0 +1,53 @@
+# Set up the Third-Person Shooter game or Test Suite 
```

Comment:
Maybe add a MD comment "DO NOT DELETE - I am a page linked to from the Third-Person Shooter game
(https://github.com/spatialos/UnrealGDKThirdPersonShooter) and Test Suite (https://github.com/spatialos/UnrealGDKTestSuite) repos. I am not linked to from the ToC as I am only relevant to these repos - coz otherwise some do-gooder will come along and think "it's not linked from the ToC."